src/components/pages/search-pages/subject-search/SubjectSearchPage.tsx

```diff
@@ -65,20 +67,20 @@ const SubjectSearchPage = () => {
       {data && <SubjectSearchList subjects={data.subjects} />}
       {isLoading ||
         (isFetching && (
-          <div className={styles['page-loader']}>
+          <Box sx={stylesSearch.pageLoader}>
             <Loader size={LoaderSize.SMALLEST} />
-          </div>
+          </Box>
         ))}
       {data?.subjects?.length === (curPage + 1) * pageSize && (
         <Button
-          className={styles['load-btn']}
+          sx={stylesPage.loadBtn}
           text="Завантажити ще"
           variant={ButtonVariant.FILLED}
           color={ButtonColor.SECONDARY}
           onClick={() => setCurPage(pr => pr + 1)}
```

Comment:
move this to the separate callback and rename `pr` to `prev`

Reply:
should I use hook useCallback here?

Reply:
should I use hook useCallback here?